### PR TITLE
Fix pagination for CLI command

### PR DESCRIPTION
**Pull request by zeineb**

### CODE REVIEW

**Comment by paraita** [comment]
Can you squash the 2 commits with the same message please ?

File: rest/rest-cli/src/test/java/org/ow2/proactive_grid_cloud_portal/cli/CommandSetTest.java [hunk 1]
```diff
@@ -38,86 +40,98 @@
...
+	@Test
+	public void testThatArgNamesMatchNumberOfArgs() throws ParseException, IllegalAccessException {
+		int nbArgsBasedOnName = Option.UNINITIALIZED;
+		String regex = "\\[.*\\]";
+		Pattern pattern = Pattern.compile(regex);
+		String optionalArgNames = "";
+		int optionals = 0;
+
+		if (entry.argNames() != null) {
+			String argNames = entry.argNames();
```
Comment (sophiesongge) [inline rest/rest-cli/src/test/java/org/ow2/proactive_grid_cloud_portal/cli/CommandSetTest.java]: This file is reformed, it's hard to see where is changed. By the way, for the format of code, we need to apply the ActiveEon Coding Rule: https://github.com/ow2-proactive/coding-rules if not the build will failed.

**Comment by marcocast** [comment]
retest this please

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 4 issues

1. MAJOR [ListJobCommand.java#L50](https://github.com/ow2-proactive/scheduling/blob/84765849b557429644d714cb78d8559038f885f4/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java#L50): Remove this unused "offsetFlag" private field. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1068)
1. MINOR [ListJobCommand.java#L35](https://github.com/ow2-proactive/scheduling/blob/84765849b557429644d714cb78d8559038f885f4/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java#L35): Remove this unused import 'org.ow2.proactive_grid_cloud_portal.cli.RestConstants'. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)
1. MINOR [ListJobCommand.java#L37](https://github.com/ow2-proactive/scheduling/blob/84765849b557429644d714cb78d8559038f885f4/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java#L37): Remove this unused import 'org.ow2.proactive_grid_cloud_portal.cli.cmd.AbstractIModeCommand'. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)
1. MINOR [ListJobCommand.java#L110](https://github.com/ow2-proactive/scheduling/blob/84765849b557429644d714cb78d8559038f885f4/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java#L110): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

**Comment by marcocast** [comment]
retest this please

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 4 issues

1. MAJOR [ListJobCommand.java#L50](https://github.com/ow2-proactive/scheduling/blob/69396d20674d5008aa834b7c18219bb7182ef426/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java#L50): Remove this unused "offsetFlag" private field. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1068)
1. MINOR [ListJobCommand.java#L35](https://github.com/ow2-proactive/scheduling/blob/69396d20674d5008aa834b7c18219bb7182ef426/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java#L35): Remove this unused import 'org.ow2.proactive_grid_cloud_portal.cli.RestConstants'. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)
1. MINOR [ListJobCommand.java#L37](https://github.com/ow2-proactive/scheduling/blob/69396d20674d5008aa834b7c18219bb7182ef426/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java#L37): Remove this unused import 'org.ow2.proactive_grid_cloud_portal.cli.cmd.AbstractIModeCommand'. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)
1. MINOR [ListJobCommand.java#L110](https://github.com/ow2-proactive/scheduling/blob/69396d20674d5008aa834b7c18219bb7182ef426/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java#L110): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

**Comment by marcocast** [comment]
retest this please

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MINOR [ListJobCommand.java#L104](https://github.com/ow2-proactive/scheduling/blob/e2d558eec983e3aa04454eb48be2889364a53fe4/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java#L104): Define and throw a dedicated exception instead of using a generic one. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)

**Pull request merged**